### PR TITLE
new og default image from Kelly

### PR DESCRIPTION
**Pull request by kimlarocca**
also added a default alt tag

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -73,10 +74,7 @@ onMounted(() => {
           name="twitter:description"
           content="Investigating a strange world."
         />
-        <Meta
-          name="twitter:image"
-          content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png"
-        />
+        <Meta name="twitter:image" content="/images/default-og-image.png" />
```
Comment (bonomite) [inline layouts/default.vue:77]: need to be content="/og-default-image.png" ... to match line 54?

File: layouts/default.vue [hunk 2]
```diff
@@ -51,9 +51,10 @@ onMounted(() => {
           content="Radiolab: Podcasts | WNYC Studios | Podcasts"
         />
         <Meta name="og:description" content="Investigating a strange world." />
+        <Meta name="og:image" content="/og-default-image.png" />
         <Meta
-          name="og:image"
-          content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png"
+          name="og:image:alt"
+          content="Radiolab: Podcasts | WNYC Studios | Podcasts"
```
Comment (bonomite) [inline layouts/default.vue:57]: not sure what og:image:alt is... but should the content be an image? not title text?

File: layouts/default.vue (hunk 1 above)
Comment (kimlarocca) [reply layouts/default.vue:77]: yes duh 🙃

File: layouts/default.vue (hunk 2 above)
Comment (kimlarocca) [reply layouts/default.vue:57]: it's just a default image alt tag - I was testing on facebook and it warned me that it was missing

Comment (bonomite) [reply layouts/default.vue:57]: ahh cool

**Comment by bonomite** [comment]
image

**Review by bonomite — APPROVED** [review]
Finally! 😂

**Pull request merged**